partner_statement/report/activity_statement.py

```diff
@@ -16,6 +16,9 @@ class ActivityStatement(models.AbstractModel):
     _description = "Partner Activity Statement"
 
     def _initial_balance_sql_q1(self, partners, date_start, account_type):
+        excluded_accounts_ids = tuple(
+            self.env.context.get("excluded_accounts_ids", [])
+        ) or (-1,)
```

Comment:
```suggestion
        )
```
-1 is treated like 1

Reply:
Thanks for all the suggestions, I should be able to work on this maybe next week.
In the meanwhile, could you please clarify why 
> -1 is treated like 1

?
I thought this _(-1, ) trick_ was working well